server/src/test/java/org/apache/druid/segment/indexing/granularity/UniformGranularityTest.java

```diff
@@ -358,6 +359,18 @@ public void testUniformGranularitySpecWithLargeNumberOfIntervalsDoesNotBlowUp()
     Assert.assertEquals(3600 * 24 * 365 * 10 + 3 * 24 * 3600, count);
   }
 
+  @Test(expected = DruidException.class)
```

Comment:
For junit 5.x.x compatibility, use `Assert.assertThrows()` with the `DruidExceptionMatch`

Reply:
Done, thanks